source/utilities.tex

```diff
@@ -6705,6 +6712,16 @@
     shared_ptr<T> allocate_shared(const A& a, size_t N,
                                   const remove_extent_t<T>& u);                 // \tcode{T} is \tcode{U[]}
 
+  template<class T>
+    shared_ptr<T> make_shared_default_init();                                   // \tcode{T} is not \tcode{U[]}
+  template<class T>
+    shared_ptr<T> allocate_shared_default_init(const A& a);                     // \tcode{T} is not \tcode{U[]}
```

Comment:
"A" is not a declared identifier here. (This is a bug in the ingress paper.) The correct declaration is in [util.smartptr.shared.create].

Reply:
fixed